pnpm-lock.yaml

```diff
@@ -5912,8 +5961,18 @@ packages:
     resolution: {integrity: sha512-0KpjqXRVvrYyCsX1swR/XTK0va6VQkQM6MNo7PqW77ByjAhoARA8EfrP1N4+KlKj8YS0ZUCtRT/YUuhyYDujIQ==}
     dev: true
 
-  /lodash.pick@4.4.0:
```

Comment:
This removes a newer version of pick and adds a bunch of older versions.  Is that what you want?

Reply:
> There is no new version of lodash.pick to override to...Sharp seemed to resolve this for themselves by downgrading lodash.pick to 3.x.